src/main/java/com/example/notificationservice/service/SqsListenerService.java

```diff
@@ -0,0 +1,76 @@
+package com.example.notificationservice.service;
+
+import com.example.notificationservice.model.Emprunt;
+import com.example.notificationservice.notification.ConfirmationNotification;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.services.sqs.model.Message;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+import software.amazon.awssdk.services.sqs.SqsClient;
+import software.amazon.awssdk.services.sqs.model.ReceiveMessageRequest;
+import software.amazon.awssdk.services.sqs.model.DeleteMessageRequest;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class SqsListenerService {
+
+    private final SqsClient sqsClient;
+    private final ObjectMapper objectMapper;
+    private final EmailService emailService;
+    public String queueUrl = "https://sqs.eu-north-1.amazonaws.com/774305596814/notification-sqs";
+
+    public SqsListenerService(SqsClient sqsClient, EmailService emailService) {
+        this.sqsClient = sqsClient;
+        this.emailService = emailService;
+        this.objectMapper = new ObjectMapper();
+    }
+
+    private void deleteMessageFromQueue(Message message) {
```

Comment:
private methods should be always in the bottom of your class, after public ones!